lib/solve.cpp

```diff
@@ -394,18 +395,6 @@ namespace quda
       x[0] = ColorSpinorField(cudaParam);
     }
 
-    if (param.use_init_guess == QUDA_USE_INIT_GUESS_YES && !param.chrono_use_resident) { // download initial guess
```

Comment:
Why are we deleting this chunk of code?

Reply:
Accident as part of the merge of conflicted branches.  Thanks for catching this.